nengo_gui/page.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, gui, filename, reset_cfg=False):
         self.finished = False  # should this Page be shut down
         self._sim = None       # the current nengo.Simulator
         self.rebuild = False   # should the model be rebuilt
+        self.sims_to_close = []  # list of sims that should be closed
```

Comment:
Even if it's made to be a set, that won't stop a Simulator from maybe being closed twice, if it gets added twice at different times (i.e. if gets added to ```sims_to_close```, then it gets closed, and then it gets added to ```sims_to_close``` again).  Indeed, I'm having trouble coming up with scenarios where it's even possible to get up to a length of 2 on this list... So my feeling is that doing this as a set is overkill.  :)